content/project/trademark/index.adoc

```diff
@@ -71,3 +95,26 @@ attributions difficult. When made necessary by space limitations, any
 reasonable facsimile of these attributions may be used. In case of doubt as to
 the proper shortened form, examples may be submitted to the Jenkins project for
 approval.
+
+== References
+
+* link:https://trademarks.justia.com/854/47/jenkins-85447465.html[Jenkins - Trademark Details #4664929]
+* link:https://www.linuxfoundation.org/trademark-usage/[trademark usage guidelines] by The Linux Foundation
+* link:approved-usage[Approved Trademark Usage]
+* link:sublicense[Trademark Sublicense Agreement]
+
+== Change History
+
+=== Oct 14, 2020
+
+The link:/project/governance-meeting[Jenkins Governance Meeting] reviewed the trademark usage policy
+and voted for closer alignment with the link:https://www.linuxfoundation.org/trademark-usage/[trademark usage guidelines] defined by The Linux Foundation.
+
+* We adopt Linux Foundation pre-approved trademark patterns and usage guidelines.
+  As long as the naming patterns are followed, no approval from the project is needed.
+* We are not applying requirements retrospectively, all link:approved-usage[previously approved trademark usages] remains approved.
+* Until the transition to CDF is over, we may make exceptions in the naming policy.
+  Example: precedent in already approved trademarks.
```

Comment:
It's not precedent if it followed the rules at the time.

We're just grandfathering in previous approvals, RHEL style (IIUC).

In fact, there is currently no approved trademark usage since we started moving to CDF/LF that would be considered an exception.